include/groonga/db.h

```diff
@@ -42,6 +42,24 @@ GRN_API void
 grn_db_touch(grn_ctx *ctx, grn_obj *db);
 GRN_API grn_rc
 grn_db_recover(grn_ctx *ctx, grn_obj *db);
+/**
+ * \brief Unmaps all opened tables and columns in the passed database.
+ *        Resources used by these opened tables and columns are freed.
```

Comment:
How about adding a new line here for readability?

```suggestion
 *
 *        Resources used by these opened tables and columns are freed.
```

Reply:
I don't have a line break because I think it is a set description with the previous sentence, but would it be easier to understand if there was a line break?

Reply:
Ahh I ses. I think it's good as it is,